src/main/java/com/bootcamp/demo/models/Booking.java

```diff
@@ -0,0 +1,102 @@
+package com.bootcamp.demo.models;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+import java.util.UUID;
+
+
+public final class Booking {
+    private UUID id;
+    private UUID serialNumber;
+    private UUID accountId;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+    private PaymentStatus payment;
+
+    public Booking() {
+    }
+
+    public Booking(UUID id, UUID serialNumber, UUID accountId, LocalDateTime startDate, LocalDateTime endDate, PaymentStatus payment) {
```

Comment:
The booking id needs to be a random generated UUID,therefore you don't need to include to the constructor parameter list.

Reply:
Also, as pointed out by @mariatirlea on slack, we could have the fields/params which are now UUID as String, and rely on other classes to do the generation as appropriate (here assign a UUID's string value). What do you think?